doc/terminology.md

```diff
@@ -0,0 +1,86 @@
+# Terminology
+
+This pages gives an overview about the terms we use, when talking about KUKSA.val components, architectures and the data handled by KUKSA.val.
```

Comment:
I suggest we add a disclaimer/note that KUKSA.val Server is not included in the description

Reply:
Put some words to that effect in the beginning, when introducing (VSS resource) Server. I do hope most aspects in this document are generic, and try only to point to databroker as “example” or “further reading” how it is done by KUKSA.val